### ISSUE TITLE
Error message is a list of numbers

### ISSUE THREAD

**Issue by geelen** [vapor/toolbox#100]
Trying to create a new project from a different template (and getting the syntax wrong) I got the following error message:

```
> vapor new Light --template https://github.com/vapor/light-template
Error: backgroundExecute(6, [], [39, 48, 48, 48, 10, 39])
```

Guessing that's not the desired behaviour :)

**Comment by tannernelson** [comment]
Yup, that should be getting converted to a string first.

**Comment by codesman** [comment]
@tannernelson I have been looking at this and thought I'd let you know what I found.

This behavior occurs only in the release build but it appears to be caused by a problem that exists in the debug build as well. In the debug build it fails silently and clones the wrong repo.

The problem appears to be in the `loadTemplate` method in `New.swift`. When the template option is passed with incorrect syntax, the options parser in `Console+Options.swift` parses this as 3 arguments versus 2 when the correct syntax is used.  In this case, the 3rd argument is `--template`. This is parsed by the options parser as `--template=true`, so when we ask for `options["template"]?.string` we get back a string of the boolean `true`.

Now that we have assigned `template` the value of `true`, it gets passed to the `expand` method, which returns `https://github.com/vapor/true-template/`

And then finally when it tries to clone that non-existent repo, we get a fatal error `repository not found`

I have a solution that fixes this and will PR, but I'm not sure it is how it should be fixed or not.
